homeassistant/helpers/json.py

```diff
@@ -41,6 +41,8 @@ def json_encoder_default(obj: Any) -> Any:
         return obj.as_dict()
     if isinstance(obj, Path):
         return obj.as_posix()
+    if isinstance(obj, NamedTuple):
+        return tuple(obj)
```

Comment:
This is not covered by tests because `isinstance(obj, (set, tuple))` will match on namedtuples

Reply:
It works better if I change it in the right place.